spec/suse/connect/api_spec.rb

```diff
@@ -77,7 +77,8 @@
   let(:expected_request_headers) do
     {
       'Authorization' => 'Basic ' + Base64.encode64("#{username}:#{password}").strip,
-      'User-Agent' => "RMT/#{RMT::VERSION}"
+      'User-Agent' => "RMT/#{RMT::VERSION}",
+      'RMT' => api_client.send(:system_uuid)
```

Comment:
Declare some test value in a let block, e.g. `let(:uuid) { 'test-uuid' }` and expect it here: `RMT => uuid`.